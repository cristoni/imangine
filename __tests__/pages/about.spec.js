import React from 'react';
import About from '../../pages/about';
import renderer from 'react-test-renderer';

test('About renders the about page', () => {
  const component = renderer.create(
    <About />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});