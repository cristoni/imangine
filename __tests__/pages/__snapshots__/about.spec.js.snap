// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders the about page 1`] = `
<p>
  about
</p>
`;
